packages/opa-react/src/authz-provider.tsx

```diff
@@ -43,6 +49,8 @@ export type AuthzProviderContext = {
    * If unset, any non-undefined, non-false (i.e. "truthy") result will be taken to mean "authorized".
    */
   defaultFromResult?: (_?: Result) => boolean;
+
+  queryClient: QueryClient;
```

Comment:
`SDK` doesn't fit well as the interface name, perhaps `OPAClient` fits better (from the comment one line above)